python/servo/build_commands.py

```diff
@@ -291,14 +291,10 @@ def build(self, release=False, dev=False, jobs=None, params=None,
 
             # Ensure that the NuGet ANGLE package containing libEGL is accessible
             # to the Rust linker.
-            append_to_path_env(
-                path.join(
-                    os.getcwd(), "support", "hololens", "packages",
-                    "ANGLE.WindowsStore.Servo.2.1.13", "bin", "UAP", arch['angle']
-                ),
-                env,
-                "LIB"
-            )
+            append_to_path_env(angle_root(self, target_triple),
+                               env,
+                               "LIB"
+                               )
```

Comment:
nit: this can not fit on one line - `append_to_path_env(angle_root(target_triple), env, "LIB")`